README.md

```diff
@@ -1,9 +1,11 @@
-# Northcoders News API
+# NC News Server
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Set-up instructions
 
+## To run this
 
+Create .env files .env.test and .env.development in the root of the folder
 
---- 
+Add DB name (check SQL set up)
```

Comment:
Which DB name should go to which file? Where shoudl they look when you mention the SQL set up? It's worth being more explicit here so there isn't any room for confusion for someone trying to run your project locally :)

Reply:
Thanks Modou, i will add some further detail to the readme :)